Packs/FeedZoom/Integrations/FeedZoom/FeedZoom_test.py

```diff
@@ -0,0 +1,16 @@
+from FeedZoom import Client
+
+
+URL = "https://support.zoom.us/hc/en-us/articles/201362683-Network-Firewall-or-Proxy-Server-Settings-for-Zoom"
+
+
+def test_build_iterator():
```

Comment:
The practice of unit testing in XSoar should not include any http calls.
Since you already imported a mock file for testing, it would be better to mock the http call to return the file's value.
If you need any help doing it, please ask.